server/src/util/jest-db.ts

```diff
@@ -0,0 +1,52 @@
+import { Model, ModelCtor, Sequelize } from 'sequelize'
+import { emailValidator } from '../bootstrap/email-validator'
```

Comment:
yea don't bring this in, since it has a dependency on `authConfig`

Reply:
replaced emailvalidator with one made for test